Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -43,6 +46,11 @@ public class MatchController : MonoBehaviour
     [SerializeField]
     private int rewardBase = 1;
 
+    public Timer roundTimer;
+
+    [SerializeField]
+    private GlobalHUDController globalHUDController;
```

Comment:
Since this component is on the same game object  consider adding a `[requireComponent()]` field and so referencing that through a `GetComponent()` call.

Reply:
I disagree for multiple reasons:

- Since this is in the scope of MatchController, this would have to be a getComponentInchildren or have a refference to the HUD object child which defeat the purpose.
- The matchmanager might find itself in situations where it doesn't even need the global HUD (like being in the bidding-scene).
- GetComponent is nice to avoid when it makes sense. MatchController itself being a required component on the MatchManager itself already implies that this script is prefab specific, meaning people are unlikely to find the requiredComponent useful.